jest.config.mjs

```diff
@@ -1,18 +1,21 @@
-const path = require('path');
-const { pathsToModuleNameMapper } = require('ts-jest');
-const { compilerOptions } = require('./tsconfig');
+import path from 'node:path';
+import url from 'node:url';
+import { pathsToModuleNameMapper } from 'ts-jest';
+import tsconfigJSON from './tsconfig.json' assert { type: "json" };
```

Comment:
This causes experimental warnings unfortunately. However that's ok, it should probably be ready eventually.